### PR TITLE
EIP-141: Designated INVALID instruction

### PR DESCRIPTION
**Pull request by gasolin**
https://github.com/ethereum/EIPs/blob/master/EIPS/eip-141.md

Since all unknown OP code will fallback to `INVALID` pseudo instruction
https://github.com/ethereumjs/ethereumjs-vm/blob/master/lib/opcodes.js#L156

and `runCode` already handle the `opName === 'INVALID'` case, 
https://github.com/ethereumjs/ethereumjs-vm/blob/master/lib/runCode.js#L119

it's a 2 lines patch

Do i need to enable or write certain test cases to test this?

### CODE REVIEW

File: lib/opcodes.js [hunk 1]
```diff
@@ -149,11 +149,12 @@ const codes = {
   0xf4: ['DELEGATECALL', 700, 6, 1, true],
 
   // '0x70', range - other
+  0xfe: ['INVALID', 0, 0, 0, true],
   0xff: ['SELFDESTRUCT', 5000, 1, 0, false]
 }
 
 module.exports = function (op, full) {
-  var code = codes[op] ? codes[op] : ['INVALID', 0]
+  var code = codes[op] ? codes[op] : ['INVALID', 0, 0, 0, true]
```
Comment (axic) [inline lib/opcodes.js]: Why is the `dynamic` flag set to true?

**Comment by gasolin** [comment]
updated to `false`, how can I know if the opcode is not dynamic?

**Comment by axic** [comment]
It is not used in the VM, but refers to state read access.

**Comment by axic** [comment]
Just to leave the reasoning here: this patch is not changing anything, but is useful if an assembler uses the same `opcodes.js` because it can emit an `INVALID` mnemonic.

**Pull request merged**